src/java.base/share/classes/java/util/concurrent/StructuredTaskScope.java

```diff
@@ -344,977 +374,700 @@ public sealed interface Subtask<T> extends Supplier<T> permits SubtaskImpl {
         enum State {
             /**
              * The subtask result or exception is not available. This state indicates that
-             * the subtask was forked but has not completed, it completed after the task
-             * scope was {@linkplain #shutdown() shut down}, or it was forked after the
-             * task scope was shut down.
+             * the subtask was forked but has not completed, it completed after the scope
+             * was cancelled, or it was forked after the scoped was cancelled (in which
+             * case a thread was not created to execute the subtask).
              */
             UNAVAILABLE,
             /**
-             * The subtask completed successfully with a result. The {@link Subtask#get()
-             * Subtask.get()} method can be used to obtain the result. This is a terminal
-             * state.
+             * The subtask completed successfully. The {@link Subtask#get() Subtask.get()}
+             * method can be used to get the result. This is a terminal state.
              */
             SUCCESS,
```

Comment:
We should use all adjectives or all nouns for the enum constants.  Since `UNAVAILABLE` is too hard to convert to a noun, we can consider renaming `SUCCESS` to `SUCCESSFUL`.

https://mail.openjdk.org/pipermail/loom-dev/2024-December/007361.html

Reply:
For context, the current name is chosen for parity with [Future.State].(https://github.com/openjdk/jdk/blob/24c5ff7ba58cb7cf93df07f81484cd8fae60e31e/src/java.base/share/classes/java/util/concurrent/Future.java#L251)